### DISCUSSION TITLE
Seeking for some help about the LRANGE and LTRIM CMD

### DISCUSSION THREAD

**Discussion by zhangfeizaici** [alicebob/miniredis#187] [Q&A]
Now I'm using the miniredis in my project for test.I find that the LRANGE and LTRIM cmd is supported in the readme. But they are not among in the exported functions of miniredis server.So I have to parse the result of client.Do which is really complex.Is there any plan to support these cmd in the future? Thanks.

**Comment by alicebob** [comment]
Hi @zhangfeizaici,

if I understand it correctly, you would like to access the LRANGE and LTRIM functions directly in miniredis, without going through a redis client? Something along these lines: `m.Lrange(string, int, int) ([]string, error)` ?

**Reply by zhangfeizaici** [reply]
Yes.Now I use the LRANGE and LTRIM cmd through raw client
